tests/test_config_node.py

```diff
@@ -151,7 +151,7 @@ def test_add_cylc_install_to_rose_conf_node_opts(rose_conf, cli_conf, expect):
 
     expect_opt = ''
     if 'opts' in cli_conf:
-        expect_opt = cli_conf['opts']
+        expect_opt = cli_conf.get('opts')
```

Comment:
FYI: I think the shellcheck suggestion was to avoid the `if _ in _` check:

```suggestion
    expect_opt = cli_conf.get('opts', '')
```